cluster/gce/gci/configure-helper.sh

```diff
@@ -493,18 +493,18 @@ function create-master-audit-policy {
   # Known api groups
   local -r known_apis='
       - group: "" # core
-      - group: "admissionregistration.k8s.io/v1alpha1"
-      - group: "apps/v1beta1"
+      - group: "admissionregistration.k8s.io"
```

Comment:
Oops! I'm going to send this as a separate PR, because we might try and cherrypick it to 1.7